go/private/actions/link.bzl

```diff
@@ -165,7 +165,7 @@ def _bootstrap_link(go, archive, executable, gc_linkopts):
         mnemonic = "GoLink",
          # workaround: go link tool needs some features of gcc to complete the job on Arm platform.
          # So, PATH for 'gcc' is required here on Arm platform.
-        command = "export GOROOT=$(pwd)/{} && export GOROOT_FINAL=GOROOT && export PATH={} && {} {}".format(go.root, go.cgo_tools.compiler_path, go.go.path, " ".join(args)),
+        command = "export GOROOT=$(pwd)/{} && export GOROOT_FINAL=GOROOT && export PATH=\"{}\" && {} {}".format(go.root, go.cgo_tools.compiler_path, go.go.path, " ".join(args)),
```

Comment:
It would be better to use `shell.quote` from go/private/skylib/lib/shell.bzl for this. We should probably actually do this for all the strings in this command.

Reply:
Done, also used the helper on some other paths inside this function.